File: programmers/K번째수.js
/*
K번째수
https://programmers.co.kr/learn/courses/30/lessons/42748?language=javascript
*/
function solution(array, commands) {
    var answer = [];
    for(let index = 0; index < commands.length; index++) {
        let [i,j,k] = commands[index];
        answer.push(array.slice(i-1,j).sort((a,b) =>  a-b)[k-1]);
    }
    console.log(answer);
    return answer;
}

// 참고 필요 - sorting : https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

File: programmers/K번째수.py
'''
K번째수
https://programmers.co.kr/learn/courses/30/lessons/42748?language=python3
'''
def solution(array, commands):
    answer = []
    for command in commands:
        _array = sorted(array[command[0]-1:command[1]])
        answer.append(_array[command[2]-1])
    return answer